frontend/src/components/classify-failures.js

```diff
@@ -248,35 +199,25 @@ export class ClassifyFailuresTable extends React.Component {
       rows,
       selectedResults,
       includeSkipped,
-      isExceptionOpen,
-      exceptionSelections,
-      exceptions,
     } = this.state;
+    const {
+      run_id
+    } = this.props
     const pagination = {
       pageSize: this.state.pageSize,
       page: this.state.page,
       totalItems: this.state.totalItems
     }
-    // filters for the exception
-    const exceptionFilters = [
-      <React.Fragment key="value">
-        <Select
-          aria-label="exception-filter"
-          placeholderText="Filter by exception"
-          variant={SelectVariant.checkbox}
-          isOpen={isExceptionOpen}
-          selections={exceptionSelections}
-          maxHeight={"1140%"}
-          isDisabled={exceptions.length < 2}
-          onToggle={this.onExceptionToggle}
-          onSelect={this.onExceptionSelect}
-          onClear={this.onExceptionClear}
-        >
-          {exceptions.map((option, index) => (
-            <SelectOption key={index} value={option._id} description={option.count + ' results'}/>
-          ))}
-        </Select>
-      </React.Fragment>
+    // filters for the metadata
+    const resultFilters = [
+      <MetaFilter
+        key="metafilter"
+        // user_properties fields shouldn't be injected here
+        fieldOptions={['metadata.user_properties.assignee', 'metadata.user_properties.component', ...STRING_RESULT_FIELDS,]}
+        runId={run_id}
+        setFilter={this.setFilter}
+        customFilters={['result*failed;error']}
```

Comment:
Does the include `skips/xfails` checkbox factor in here?

Reply:
It does now!  Validated with some results that had skips and the checkbox works as expected now and impacts the counts of value select options.